schema/mysql/v56/cadence/schema.sql

```diff
@@ -64,17 +64,17 @@ CREATE TABLE transfer_tasks(
 
 CREATE TABLE executions(
   shard_id INT NOT NULL,
-	domain_id CHAR(64) NOT NULL,
-	workflow_id VARCHAR(255) NOT NULL,
-	run_id CHAR(64) NOT NULL,
+	domain_id BINARY(16) NOT NULL,
+	workflow_id VARCHAR(64) NOT NULL,
```

Comment:
I don't think we can limit workflow_id to 64 bytes. This is too low for some use cases.

Reply:
ok. changed it back to 255